builders/ethereum/dev-env/thirdweb.md

```diff
@@ -110,35 +112,47 @@ thirdweb offers SDKs for a range of programming languages, such as React, React
+
+Press **Create API Key** then take the following steps:
+
+1. Give your API key a name
+2. Enter the allowed domains that the API key should accept requests from. It's recommended that you allow only necessary domains, but for development purposes, you can select **Allow all domains**
+3. Press **Next** and confirm the prompt on the next page
+
+Finally, specify your Client ID (API Key) in your `client.ts` file. ThirdWeb by default assumes it will be in your `.env` file named `NEXT_PUBLIC_TEMPLATE_CLIENT_ID`.
+
+![thirdweb create API key](/images/builders/ethereum/dev-env/thirdweb/thirdweb-3.webp)
+
 ## Interact With a Contract {: #interact-with-a-contract }
 
-thirdweb provides several SDKs to allow you to interact with your contract including: [React](https://portal.thirdweb.com/react){target=\_blank}, [React Native](https://portal.thirdweb.com/react-native){target=\_blank}, [TypeScript](https://portal.thirdweb.com/typescript){target=\_blank}, and [Unity](https://portal.thirdweb.com/unity){target=\_blank}.
+thirdweb provides several SDKs to allow you to interact with your contract including: [React](https://portal.thirdweb.com/typescript/v5/react){target=\_blank}, [React Native](https://portal.thirdweb.com/typescript/v5/react-native){target=\_blank}, [TypeScript](https://portal.thirdweb.com/typescript/v5){target=\_blank}, and [Unity](https://portal.thirdweb.com/unity){target=\_blank}.
 
-This document will show you how to interact with your contract deployed to Moonbeam using React. You can view the [full React SDK reference](https://portal.thirdweb.com/react){target=\_blank} in thirdweb’s documentation.
+This document will show you how to interact with your contract deployed to Moonbeam using React. You can view the [full React SDK reference](https://portal.thirdweb.com/typescript/v5/react){target=\_blank} in thirdweb’s documentation.
 
-To create a new application pre-configured with thirdweb’s SDKs run and choose your preferred configurations:
+### Run Locally {: #run-locally }
+
+To run your dApp locally for testing and debugging purposes, use the command: 
 
 ```bash
-npx thirdweb create app --evm
+yarn dev
 ```
 
-Or install it into your existing project by running:
+The app will compile and specify the localhost and port number for you to visit in your browser.
 
-```bash
-npx thirdweb install
-```
+![thirdweb run locally](/images/builders/ethereum/dev-env/thirdweb/thirdweb-4.webp)
 
 ### Initialize SDK On Moonbeam {: #initialize-sdk-on-moonbeam }
 
 Wrap your application in the `ThirdwebProvider` component and change the `activeChain` to Moonbeam.
 
-```javascript
+```javascript title="Initialize SDK"
 import { ThirdwebProvider } from '@thirdweb-dev/react';
```

Comment:
According to the migration guide to v5, we shouldn't be using @thirdweb-dev instead we should use the SDK https://portal.thirdweb.com/typescript/v5/migrate - did you look into this?

Reply:
They have an example in the migration guide that would essentially replace the below snippet, but that is like a progressive way to migrate. I think we should just start fresh with v5 https://portal.thirdweb.com/typescript/v5/react/ThirdwebProvider. It seems that we wouldn't really need to do anything with the app to create the ThirdwebProvider but we would have to look into how to use Moonbeam https://portal.thirdweb.com/typescript/v5/chain